README.md

```diff
@@ -40,10 +40,20 @@ CMR-GraphQL uses a few environment variables for configuring runtime options:
 |MMT_ROOT_URL||URL to ping when retrieving metadata from MMT e.g. https://mmt.earthdata.nasa.gov|
 |DRAFT_MMT_ROOT_URL||URL to ping when retrieving draft metadata from Draft MMT e.g. https://draftmmt.earthdata.nasa.gov|
 |LAMBDA_TIMEOUT|30|Number of seconds to set the Lambda timeout to.|
-|EDL_KEY_ID, EDL_JWK, EDL_CLIENT_ID|For facilitating EDL connection -- obtain these from a dev|
+|EDL_KEY_ID, EDL_JWK, EDL_CLIENT_ID, EDL_PASSWORD| .bash or .zshrc file|For facilitating EDL connection -- obtain these from a dev and upload. After making changes, remember to run $ source ~/.zshrc|
 
-### Serverless Framework
+### Local Environment Spin-Up
+If using cmr-graphql in conjunction with MMT, setup local CMR database with the [instructions found here](https://github.com/nasa/mmt/tree/main). Then utilize one of the following commands depending on the 
```

Comment:
MMT specific help should exist in the MMT readme. Adding this information here will only add confusion for people who aren't developing MMT.